whatsnew/src/main/java/io/github/tonnyl/whatsnew/util/ItemLayoutOption.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.tonnyl.whatsnew.util
+
+/**
+ * Created by lizhaotailang on 30/11/2017.
```

Comment:
🤔

Reply:
Sorry i just duplicate the same class presentationOptions , can i put my name in the created class?

Reply:
Of course you can 🎉